tests/models/models_test.py

```diff
@@ -32,6 +32,24 @@ def test_set_sample_rate_raises_warning():
         model.sample_rate = 16000.0
 
 
+def test_multichannel_model_loading():
+    class MCModel(BaseModel):
+        def __init__(self, sample_rate=8000.0, n_channels=2):
```

Comment:
As said in the PR. We must have the `sample_rate` as argument. 

If we want fixed number of channels, we don't need it in the `__init__`, we just `super(n_channels=2)` and that works.